src/Service/datastore-service.ts

```diff
@@ -0,0 +1,9 @@
+import Datastore from 'react-native-local-mongodb';
```

Comment:
I recommend using [WatermelonDB](https://github.com/Nozbe/WatermelonDB)

Reply:
`react-native-local-mongodb` is overblowed. Don't use this.

Reply:
currently we are using asyncstorage